app/views/questions/_form.html.erb

```diff
@@ -8,6 +8,6 @@
     </div>  
   </div>
   <div class="col-xs-3 col-sm-2">
-    <button type="submit" class="btn btn-primary"><i class="fa fa-reply visible-xs"></i><span class="hidden-xs">Submit</span></button>
+    <button type="submit" id="submit-comment-button" class="btn btn-primary"><i class="fa fa-spinner fa-spin hidden" id="loading-spinner"></i><i class="fa fa-reply" id="reply-comment"></i><span id="submit-comment"> Submit</span></button>
```

Comment:
You should continue using the `visible-xs` and `hidden-xs` classes as this was done to adjust the button with the box for smaller screen widths.